src/utils/styled.js

```diff
@@ -3,8 +3,8 @@ const isTaggedTemplateLiteral = require('./tagged-template-literal').isTaggedTem
 /**
  * Check if something is a styled-components import declaration
  */
-const isStyledImport = node =>
-  node.type === 'ImportDeclaration' && node.source.value === 'styled-components'
+const isStyledImport = (node, importName) =>
+  node.type === 'ImportDeclaration' && node.source.value === importName
```

Comment:
Hmm in the README you're saying this depends on the name you import it with, but here we check the library imported, so you should probably update that in the README. Our code is actually agnostic to the variable names you assign them to, as far as I understand anyway. So therefore if you want to use it with another library you would have to specify that option, as we never check for variables called `styled` I believe.

In that case the argument here and the option name should probably be `libraryName`, yeah?

Reply:
Read the associated issue, I just phrased the README badly.

We default to `styled` for the assigned variable name of the default export, so it'll work if you do `import styled from 'emotion'`, and we also check the imports so it works if you do `import other from 'styled-components'`, but at the moment if you do `import other from 'emotion'` it'll break. That's what this PR fixes, basically.

Reply:
I remembered why I called it `importName` added my reasoning in the above comment. I can see why `importName` could be confusing, since "name" doesn't unambigously point to either parts of the import statement.

Babel calls it `importNode.source.value`, so maybe `importSourceName`?

Reply:
Maybe I'm thinking this through too much :thinking:

Reply:
Hmm okay, I think I get you now. My main points of misunderstanding was mostly from this line in the README:
> If you want to avoid edge cases when you assing the default import to a variable not called "styled"

As I think that's super misleading as it actually has nothing to do with what you name your the variable you import it into (it would also work, as I know now you didn't have an understanding problem with, if you did `import wabalubadubdub from 'styled-components`). 

And then I also had a problem because I never heard of the webpack aliasing thing because I never used it, when you talked about aliasing I thought you might've meant `import { injectGlobal as some_alias }` or stuff like that which again wouldn't have been relevant but I researched the webpack alias thing, and get you now!